mode/examples/Topics/Simulate/SoftBody/SoftBody.pyde

```diff
@@ -47,6 +47,7 @@ def draw():
 
 
 def drawShape():
+    global nodeStartX,nodeStartY,rotAngle
```

Comment:
nodestartX and nodeStartY do not need to be declared global because they're not being reassigned.